src/components/App.js

```diff
@@ -72,7 +82,8 @@ const App = React.createClass({
           splitterSize: 1,
           endPanelControl: true,
           startPanel: this.renderCenterPane(this.props),
-          endPanel: RightSidebar()
+          endPanel: RightSidebar(),
```

Comment:
name `RightSidebar` may be worth changing because in the vertical mode it's no longer on the right